crates/core_arch/src/aarch64/neon.rs

```diff
@@ -232,6 +242,198 @@ extern "C" {
     ) -> int8x16_t;
 }
 
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(addp))]
+pub unsafe fn vpaddq_u8(a: uint8x16_t, b: uint8x16_t) -> uint8x16_t {
+    vpaddq_u8_(a, b)
+}
+
+//poly128_t vmull_p64 (poly64_t a, poly64_t b)
+#[inline]
+#[target_feature(enable = "neon")]
+#[cfg_attr(test, assert_instr(pmull))]
+pub unsafe fn vmull_p64(a: poly64_t, b: poly64_t) -> poly128_t {
+    transmute(vmull_p64_(transmute(a), transmute(b)))
+}
+
+macro_rules! aarch64_simd_2 {
+    ($name:ident, $type:ty, $simd_fn:ident, $intr:ident) => {
+        #[inline]
```

Comment:
Missing documentation comment

Reply:
This is a template and in itself won't have documentation - it's still missing in `aarch64_simd_ceq` - I'll ad it there.

Reply:
Sorry I got that wrong I think you're meaning some macro level docs, I'll add them!

Reply:
What I meant is that the macro should take a `$doc:meta` argument, and the "template" should expand it like `#[$doc]`.

Reply:
Oh I didn't knew that that worked! I'll update the macros for that, thanks!